src/Components/styled-components/StyledButton.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const StyledButton = styled.button`
+  font-size: 1.1rem;
+  width: 100px;
+  padding: 5px 15px;
+  background-color: #6c48f2;
+  color: white;
+  border-radius: 15px;
+  border: none;
+  cursor: pointer;
+  &:disabled {
+    background-color: lightgray;
+  }
+`;
```

Comment:
why does the register button have another color than the login button. Can you give them both the same color or if you want add a hover effect.